core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -859,6 +860,7 @@ public function loadDetails()
 		$this->groups = $this->protected ? StringUtil::deserialize($this->groups) : false;
 		$this->layout = $this->includeLayout ? $this->layout : false;
 		$this->cache = $this->includeCache ? $this->cache : false;
+		$this->alwaysLoadFromCache = (bool) $this->alwaysLoadFromCache;
```

Comment:
You must still check for `$this->includeCache` here, otherwise the respective checkbox could have been removed but the database entry would still affect the page settings.

Reply:
True, fixed in 16ec76d379df8302857b2e0e40f8d405a2f7c39b.